README.md

```diff
@@ -17,41 +17,68 @@ Answer the following data queries. Keep track of the SQL you write by pasting it
 ### **Clicking the `Restore Database` button in the page will repopulate the database with the original data and discard all changes you have made**.
 
 ### find all customers that live in London. Returns 6 records.
-> This can be done with SELECT and WHERE clauses
+> SELECT *
+FROM customers
+WHERE city = 'London'
 
 ### find all customers with postal code 1010. Returns 3 customers.
-> This can be done with SELECT and WHERE clauses
+> SELECT * 
+FROM customers 
+WHERE postalcode = 1010
 
 ### find the phone number for the supplier with the id 11. Should be (010) 9984510.
-> This can be done with SELECT and WHERE clauses
+>  SELECT phone
+ FROM suppliers
+ WHERE SupplierID = 11
 
 ### list orders descending by the order date. The order with date 1997-02-12 should be at the top.
-> This can be done with SELECT, WHERE, and ORDER BY clauses
+> SELECT orderdate
+FROM orders
+ORDER BY orderdate DESC
 
 ### find all suppliers who have names longer than 20 characters. You can use `length(SupplierName)` to get the length of the name. Returns 11 records.
-> This can be done with SELECT and WHERE clauses
+> TSELECT * 
+FROM suppliers
+WHERE length(suppliername)>20
 
 ### find all customers that include the word "market" in the name. Should return 4 records.
-> This can be done with SELECT and a WHERE clause using the LIKE keyword
+> SELECT * 
+FROM customers
+WHERE customername like'%market%'
 
 > Don't forget the wildcard '%' symbols at the beginning and end of your substring to denote it can appear anywhere in the string in question
 
 ### add a customer record for _"The Shire"_, the contact name is _"Bilbo Baggins"_ the address is _"1 Hobbit-Hole"_ in _"Bag End"_, postal code _"111"_ and the country is _"Middle Earth"_.
-> This can be done with the INSERT INTO clause
+>INSERT INTO CUSTOMERS (customername, contactname, address, postalcode, country)
+VALUES ('The Shire', 'Bilbo Baggins', '1 Hobbit-Hole in Bag end','111', 'Middle Earth')
+
 
 ### update _Bilbo Baggins_ record so that the postal code changes to _"11122"_.
-> This can be done with UPDATE and WHERE clauses
+> UPDATE customers
+SET postalcode = 11122
+WHERE contactname = 'Bilbo Baggins'
 
 ### list orders grouped by customer showing the number of orders per customer. _Rattlesnake Canyon Grocery_ should have 7 orders.
-> This can be done with SELECT, COUNT, JOIN and GROUP BY clauses. Your count should focus on a field in the Orders table, not the Customer table
+> SELECT  customername, COUNT(orderid) AS OrdersPerCustomer
```

Comment:
Since this request does not explicitly mention returning anything else aside from the order count, you can perform this query without having to use a `JOIN` clause (though this doesn't make your current query incorrect, since it does add clarity to the results of the original request):
```
SELECT CustomerID, COUNT(OrderID) OrderCount
FROM Orders
GROUP BY CustomerID
```